pkg/solana/txm/txm.go

```diff
@@ -568,6 +550,82 @@ func (txm *Txm) Enqueue(accountID string, tx *solanaGo.Transaction, txCfgs ...Se
 	return nil
 }
 
+func (txm *Txm) EstimateComputeUnitLimit(ctx context.Context, tx *solanaGo.Transaction) (uint32, error) {
```

Comment:
Would be nice to add comments for this method and `simulateTx` to summarize their usage to continue the good documentation in this package

Reply:
Good point, added comments in the latest commit